### PR TITLE
la_model

### PR DESCRIPTION
**Pull request by xinao-han**
1. finished la layer
2. updated data set: split training data(eng) into train and eval
3. updated train_batches_eng: set batch_size =16; add parameter labels=labels_train

### CODE REVIEW

**Comment by xinao-han** [comment]
la_model
